src/sections/Pricing/comparison.js

```diff
@@ -140,6 +141,16 @@ h2, h5{
 .category-icon{
   margin: 1rem auto 0;
 }
+.categories{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 1rem 0;
```

Comment:
Instead of adding a top/bottom margin here, remove the top margin given to the `.category-icon`